### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by mjpost**
Clarifying documentation, plus other miscellaneous updates.

### CODE REVIEW

File: hugo/layouts/index.html [hunk 1]
```diff
@@ -50,8 +50,7 @@ <h2 class="pl-2 pb-2">Welcome to the ACL Anthology!</h2>
       </div>
       {{ end }}
       <div class="col-6 col-xl-12 mb-2">
-        <a class="btn btn-block btn-warning" href="https://github.com/acl-org/acl-anthology/issues/170">Give feedback
-        </a>
+        <a class="btn btn-block btn-warning" href="{{ "/faq/feedback.md" | relURL }}">Give feedback</a>
       </div>
```
Comment (mjpost) [inline hugo/layouts/index.html:54]: Can someone (@mbollmann?) help me with the syntax here? This doesn't work, and I've tried a number of different things without success. From [a look at the docs](https://gohugo.io/content-management/cross-references/) this should be correct, but there are distinctions that I don't understand (somewhere in the docs it mentions that some tags only work for certain types of pages, but the distinction wasn't meaningful to me).

Comment (mbollmann) [reply hugo/layouts/index.html:54]: Yeah, this is not a content file, but a layout file, so stuff like shortcodes won't work. Look at how it's done when linking to venues later on the page; something like this should work:
```
($.Site.GetPage "/faq/feedback.md").RelPermalink
```

Comment (mjpost) [reply hugo/layouts/index.html:54]: That worked, thank you!

Comment (mjpost) [reply hugo/layouts/index.html:54]: Someday I'm just going to need to bite the bullet and read the Hugo docs straight through.

**Pull request merged**